samples/templates/default-azuredeploy.json

```diff
@@ -382,6 +396,53 @@
                 "principalId": "[reference(concat('Microsoft.Web/sites/', variables('serviceName')), '2018-11-01', 'full').identity.principalId]",
                 "principalType": "ServicePrincipal"
             }
+        },
+        {
+            "name": "[variables('azureContainerRegistryName')]",
+            "type": "Microsoft.ContainerRegistry/registries",
+            "apiVersion": "2019-12-01-preview",
+            "location": "[resourceGroup().location]",
+            "comments": "Container registry for storing templates images",
+            "condition": "[parameters('enableDataConvert')]",
+            "tags": {
+              "displayName": "Container Registry",
+              "container.registry": "[variables('azureContainerRegistryName')]"
+            },
+            "sku": {
+              "name": "Basic",
+              "tier": "Basic"
+            },
+            "properties": {
+              "adminUserEnabled": "true"
+            }
+        },
+        {
+            "type": "Microsoft.KeyVault/vaults/secrets",
```

Comment:
Did we consider managed identity and Azure RBAC instead?

Reply:
Especially since the documentation for adminUser states "The admin account is designed for a single user to access the registry, mainly for testing purposes."

Reply:
Yes. We considered RBAC way to authenticate ACR. Finally we choose to use credentials for OSS and RBAC for PaaS as customers can config different registries other than ACR in OSS. Besides, users can configure a service principal credential here instead of adminUser.

Reply:
We want to keep the OSS and PaaS functionality as close to each other as possible. We need not support non-ACR registries in OSS if that makes a difference.

Reply:
Sure. Have changed credential to MI and Azure RBAC